io/include/detray/io/common/detail/file_handle.hpp

```diff
@@ -100,13 +100,10 @@ class file_handle final {
     std::fstream m_stream;
 
     /// How many files have been created? Maximum: 65'536
-    static std::size_t n_files;
-    static std::size_t n_open_files;
+    std::size_t n_files{0};
+    std::size_t n_open_files{0};
```

Comment:
These need to be static in order to count the global number of files. In #506 this will be fixed the following way:
```suggestion
    inline static std::size_t n_files{0u};
    inline static std::size_t n_open_files{0u};
```

Reply:
OK Fixed now